package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "shopify-app-node",
```

Comment:
the name of the app is from the boilerplate

Reply:
This is not the name of the app

Reply:
change the name to the name of the app to have it in sync

Reply:
Done it. Let's check